packages/node/src/lib/process-request.ts

```diff
@@ -184,7 +187,7 @@ export default function processRequest(
   // We use a fake host here because we rely on the host header which could be redacted.
   // We only ever use this reqUrl with the fake hostname for the pathname and querystring.
   // req.originalUrl is express specific, req.url is node.js
-  const reqUrl = new URL(req.originalUrl || req.url, 'https://readme.io');
+  const reqUrl = new URL(req.originalUrl || req.url || '', 'https://readme.io');
```

Comment:
is empty string really valid here, or is it just to cover the possibility of a falsey type? If it's not a real possibility here it might be better to throw an error early so that the user is notified of an issue with urls.

Reply:
yeah I think there are opportunities to rework the logic to throw better validation errors (yay TS strict mode!) but I'm trying to confine the scope of the changes in this PR to fallback values where needed and avoid any bigger logic changes